components/Buttons/tests/unit/ButtonsThemingTest.swift

```diff
@@ -216,5 +216,134 @@ class ButtonsThemingTest: XCTestCase {
     XCTAssertEqual(button.minimumSize.height, 36.0, accuracy: 0.001)
   }
 
+  func testFloatingButtonSecondaryThemeWithNoCustomThemes() {
+    // Given
+    let button = MDCFloatingButton()
+    let defaultButton = MDCFloatingButton()
+    let scheme = MDCContainerScheme()
+    let colorScheme = MDCSemanticColorScheme(defaults: .material201804)
+    let typographyScheme = MDCTypographyScheme(defaults: .material201804)
+
+    // When
+    button.applySecondaryTheme(withScheme: scheme)
+
+    // Then
+    XCTAssertEqual(button.backgroundColor(for: .normal), colorScheme.secondaryColor)
+    XCTAssertEqual(button.imageTintColor(for: .normal), colorScheme.onSecondaryColor)
+    // XCTAssertEqual(button.titleColor(for: .normal), colorScheme.onSecondaryColor)
+    XCTAssertEqual(button.shadowColor(for: .normal), defaultButton.shadowColor(for: .normal))
+    XCTAssertEqual(button.inkColor, defaultButton.inkColor)
+
+    XCTAssertEqual(button.titleFont(for: .normal), typographyScheme.button)
+
+    if let buttonShape = button.shapeGenerator as? MDCRectangleShapeGenerator {
+      let corner = MDCCornerTreatment.corner(withRadius: 0.5)
+      corner?.valueType = .percentage
+      XCTAssertEqual(buttonShape.topLeftCorner, corner)
+      XCTAssertEqual(buttonShape.topRightCorner, corner)
+      XCTAssertEqual(buttonShape.bottomRightCorner, corner)
+      XCTAssertEqual(buttonShape.bottomLeftCorner, corner)
+    } else {
+      XCTFail("Button.shapeGenerator was not a MDCRectangularShapeGenerator")
+    }
+
+    // Test remaining properties
+    XCTAssertEqual(button.elevation(for: .normal), ShadowElevation.fabResting)
+    XCTAssertEqual(button.elevation(for: .highlighted), ShadowElevation.fabPressed)
+    XCTAssertEqual(button.elevation(for: .disabled), ShadowElevation.none)
+
+    XCTAssertEqual(button.borderWidth(for: .normal), defaultButton.borderWidth(for: .normal), accuracy: 0.001)
```

Comment:
In this case, we want to be sure that a FAB has no border. Please assert that the border width is (nearly) 0.

Reply:
Addressed.